### PR TITLE
[ORT Mobile] ORT Minimal E2E CI

### PR DESCRIPTION
**Pull request by gwang-msft**
**Description**: ORT minimal E2E CI
This CI will replace the previous working ORT Minimal CI
The CI has 3 steps
1. Build a full ORT, install python wheel and use it to convert the test data from onnx format to ort format, also generate an included ops config file
2. Build only step to disable exception and include all kernels to safe-guard the --disable_exceptions option
3. Build a minimal ORT using the included ops config file from previous step, also include some models from UT, run minimal UT and use minimal onnx_test_runner to run e2e test cases. Exception is enabled in this step

**Motivation and Context**
- We do not have end to end coverage for ORT minimal

### CODE REVIEW

File: tools/ci_build/github/linux/ort_minimal/build_full_ort_and_create_ort_files.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    --skip_submodule_sync \
+    --parallel \
+    --build_wheel \
+    --skip_tests \
+    --enable_pybind \
+    --cmake_extra_defines PYTHON_INCLUDE_DIR=/opt/python/cp37-cp37m/include/python3.7m PYTHON_LIBRARY=/usr/lib64/librt.so
+
+# Install the ORT python wheel
+/opt/python/cp37-cp37m/bin/python3 -m pip install -U /build/Debug/dist/*
+
```
Comment (skottmckay) [inline tools/ci_build/github/linux/ort_minimal/build_full_ort_and_create_ort_files.sh:23]: Uninstall this once done converting just so there's no chance something downstream accidentally uses the full version from the package

File: tools/ci_build/github/linux/ort_minimal/build_minimal_ort_and_run_tests.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+/opt/python/cp37-cp37m/bin/python3 /onnxruntime_src/tools/ci_build/build.py \
+    --build_dir /build --cmake_generator Ninja \
+    --config Debug \
+    --skip_submodule_sync \
+    --build_shared_lib \
+    --parallel \
+    --minimal_build \
+    --disable_ml_ops \
+    --disable_exceptions \
+    --include_ops_by_model /home/onnxruntimedev/.test_data/models_to_exclude/ \
```
Comment (skottmckay) [inline tools/ci_build/github/linux/ort_minimal/build_minimal_ort_and_run_tests.sh:28]: Do we want to disable exceptions in this test? May make it harder to investigate any issues.

Comment (gwang-msft) [reply tools/ci_build/github/linux/ort_minimal/build_minimal_ort_and_run_tests.sh:30]: Add an additional steps to verify --disable_exceptions

---
In reply to: [489895659](https://github.com/microsoft/onnxruntime/pull/5200#discussion_r489895659) [](ancestors = 489895659)

**Review by skottmckay — DISMISSED** [review]
:shipit:

**Review by skottmckay — APPROVED** [review]

**Pull request merged**